platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/metrics/StaleMetrics.java

```diff
@@ -19,12 +19,9 @@
 import static com.swirlds.metrics.api.Metrics.INTERNAL_CATEGORY;
 
 import com.swirlds.common.context.PlatformContext;
-import com.swirlds.common.platform.NodeId;
 import com.swirlds.metrics.api.LongAccumulator;
 import com.swirlds.metrics.api.Metrics;
-import com.swirlds.platform.internal.EventImpl;
 import edu.umd.cs.findbugs.annotations.NonNull;
-import java.util.Objects;
 
 /**
```

Comment:
It appears this class is no longer used anywhere

Reply:
Deleted.